src/components/cylc/gscan/GScan.vue

```diff
@@ -191,24 +186,6 @@ export default {
     numFilters () {
       return Object.values(this.filters).flat().length
     },
-    localValueWorkflow: {
-      get () {
-        return this.modelValueWorkflow
-      },
-      set (value) {
-        // Update 'modelValueWorkflow' prop by notifying parent component's v-model for this component
-        this.$emit('update:modelValueWorkflow', value)
-      }
-    },
-    localValueState: {
-      get () {
-        return this.modelValueState
-      },
-      set (value) {
-        // Update 'modelValueState' prop by notifying parent component's v-model for this component
-        this.$emit('update:modelValueState', value)
-      }
-    },
```

Comment:
This got a bit more complicated than it needed to be. What I've done above is go back to using `v-model` on the `filters` object. Using the `v-model` directive on a component is short for binding `model-value` and adding the event listener `@update:model-value`
```diff
 <TaskFilterSelect
-  :model-value="modelValueWorkflow"
+  v-model="filters[title]"
```